magicctapipe/scripts/lst1_magic/lst1_magic_train_rfs.py

```diff
@@ -78,7 +76,7 @@ def get_events_at_random(event_data, n_events_random):
     ----------
```

Comment:
I think it has never been put in a separate module (and in an init file) because it is needed only in this file (just not to repeat the same 5 lines both for protons and for gammas). But it can be moved now if you want to put it into the documentation

Reply:
I would say no need. We can keep it like this.